packages/primitives/src/reatomSet.test.ts

```diff
@@ -0,0 +1,124 @@
+import { createCtx } from '@reatom/core'
+import { suite } from 'uvu'
+import * as assert from 'uvu/assert'
+
+import { reatomSet } from './reatomSet'
+
+const test = suite('reatomSet')
+
+test(`reatomSet. init`, () => {
```

Comment:
Why do you need reatomSet in test name of it's already in suite name?..

Reply:
I just used a template from another test in package 

![image](https://github.com/artalar/reatom/assets/45297354/8a95b4c4-4d5f-4be9-992f-28f8452a8e4b)